Assets/Localization/StringTables/Internal_RSC_en.asset

```diff
@@ -12752,7 +12752,7 @@ MonoBehaviour:
     m_Localized: 'Barbarians are warriors that hail from the harsh[/center]
 
       lands
-      that fringe more civilized areas. They make excellent[/center]
+      that fringe more-civilized areas. They make excellent[/center]
```

Comment:
English is not my native language, but I haven't seen a dash used in that context before or by looking around

Reply:
I perhaps use more hyphens than most writers, but I think it's justified in this case to show the two words are connected, so as to intuitively clarify that it shouldn't be misinterpreted as 'a greater number of civilized areas.' I dislike ambiguous syntax even if it's considered technically-correct English.